src/hooks/useActionCenterSubscriber/useLimitOrderActionSubscriber.tsx

```diff
@@ -37,21 +27,23 @@ import {
   selectLimitOrderSubmissionMetadata,
 } from '@/state/slices/limitOrderSlice/selectors'
 import {
+  selectActions,
```

Comment:
same q here re: narrowing to limit order actions with a specific selector

Reply:
Done in [c30c8e9](https://github.com/shapeshift/web/pull/9738/commits/c30c8e93ba075d595b1d8f868461c4e171aaaa02)